### ISSUE TITLE
Accessibility: A topic for the guide?

### ISSUE THREAD

**Issue by Madago765** [1DSAG/ABAP-Leitfaden#245]
Hi,

during the presentation of the topic of "Barrierefreiheit" / accessibility at the DSAG event in Hockenheim on 10/11.02.2026, I asked myself to what extent this should be taken into account in the new ABAP guideline? 
Has it ever been discussed? 
Should it be included?

Regards

MDG

**Comment by Xexer** [comment]
Hello @Madago765,
was not part of the discussion as we have heard that moat Developers are not aware of this topic. I think we have to check, how we can extend the Guideline.

Thank you for the Input.
